Project/Fall2020_CSC403_Project/FrmLevel.cs

```diff
@@ -47,9 +49,17 @@ public partial class FrmLevel : Form {
 
     private void OnEnemyDeath(Enemy enemy)
     {
-      // Render enemy off of the screen
-      enemy.Icon.Visible = false;
-
+      // Render enemy off of the screen after death animation
+      enemy.Icon.LoadAsync("C:\\Users\\jbt01\\Desktop\\LaTech fall 2020\\csc403project\\Fall2020_CSC403_Project\\Project\\Fall2020_CSC403_Project\\data\\chardeath.gif");
+      enemy.Icon.SizeMode = PictureBoxSizeMode.StretchImage;
+      // Using undrawable image to hide background image
+      enemy.Icon.BackgroundImage = global::Fall2020_CSC403_Project.Properties.Resources.playermoving;
+
+      // Play death animation then hide image    
+      deathtimer = new System.Timers.Timer(2000);
+      deathtimer.Elapsed += (sender, e) => Invoke((MethodInvoker)(() => stopanimate(sender, e, enemy)));
```

Comment:
Calling a method from another thread may cause an exception in some cases. You may want to use a try-catch block around this.

Reply:
try/catch added